docs/source/installation.rst

```diff
@@ -0,0 +1,8 @@
+Quick Install
+=================================
+
+Blah
```

Comment:
Is this rendered in the docs? Would rather not have the blah blah blah show up anywhere :-P

Reply:
You can just point them to the readme for now

Reply:
It was just temp - I'll clean it up